### PR TITLE
docs(errors): sort error list alphabetically ascending

### PR DESCRIPTION
**Pull request by Fdawgs**
Creating this as a draft first, as I had a quick question: is it worth converting the list of errors into a table composed of  `code` and `description` column headers? It would compress the data into an easier to reference format.

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
A table might prevent us to have some significant text and/or example attached to the error code.

Wdyt @jsumners ?

**Comment by jsumners** [comment]
I agree. A table would make maintaining this document more difficult than it needs to be. What this document needs to be is a place where people can `cmd+f`/`ctrl+f` the code to learn about it. As it is written, that's what this document provides.

**Comment by Fdawgs** [comment]
@mcollina  @jsumners thanks for the responses guys, have left structure as is!

**Review by mcollina — APPROVED** [review]
lgtm

**Review by jsumners — APPROVED** [review]
LGTM

File: docs/Errors.md [hunk 1]
```diff
@@ -7,22 +7,22 @@
 ### Error Handling In Node.js
 
 #### Uncaught Errors
-In Node.js, uncaught errors are likely to cause memory leaks, file descriptor leaks and other major production issues. [Domains](https://nodejs.org/en/docs/guides/domain-postmortem/) were introduced to try fix this issue, but did not. 
+In Node.js, uncaught errors are likely to cause memory leaks, file descriptor leaks and other major production issues. [Domains](https://nodejs.org/en/docs/guides/domain-postmortem/) were introduced to try to fix this issue, but did not. 
 
 Given that it is not possible to process all uncaught errors sensibly, the best way to deal with them is to [crash](https://nodejs.org/api/process.html#process_warning_using_uncaughtexception_correctly). 
 
-#### Catching Errors In Promises
+#### Catching Errors in Promises
```
Comment (jsumners) [inline docs/Errors.md:14]: See my comment https://github.com/fastify/fastify/pull/2871#discussion_r581007454

**Review by jsumners — CHANGES_REQUESTED** [review]
Changing my review pending further discussion.

File: docs/Errors.md (hunk 1 above)
Comment (Fdawgs) [reply docs/Errors.md]: Removed title style change commit in https://github.com/fastify/fastify/commit/858911aeb6e8ff37410300ddc01e764e75f8b96b

**Review by jsumners — APPROVED** [review]
LGTM

**Pull request merged**